### ISSUE TITLE
Pull to refresh

### ISSUE THREAD

**Issue by LarreaMikel** [LarreaMikel/uMatriks#3]
The Room List view does not refreshing properly. 
- When the user leaves a room, this room does not disappear from the list.

- When the user joins a room, this room does not appear properly in the list.

**Comment by delijati** [comment]
yeah i also run into that problem somehow the signals are not send

**Comment by LarreaMikel** [comment]
I see two different problems; highlight new messages in a room and sorting the roomlist (newest messages received rooms first). The first one is more  or less solved in [8658db4](https://github.com/LarreaMikel/uMatriks/commit/8658db4cbb94b9d5041f627cb6594cf25fd0c020) although we need to polish the performance. The second one is to be done.
